CONTRIBUTING_A_NEW_QUEST.md

```diff
@@ -0,0 +1,390 @@
+
+# Test
+
+Obviously, testing can be done also earlier. But at least before submitting for review quest should be tested.
+
+Typically it is done using an emulator. Note that you can set location in emulator settings rather than scrolling within StreetComplete itself.
+
+* Is quest listed?
+  * Look at the quest list in settings - is your quest appearing there? If not - see [this step](#Add quest to the list of active ones).
+  * While you are there you can disable all quests except yours for an easier testing.
+* Is it shown for expected elements?
+* Is it selected for some unwanted elements?
+* Can you tap on quest marker on open quest form?* an you fill answer as expected?
+* Can you solve the quest?
+* Is expected tagging being applied?
+** You can look at logs for info what was applied or use undo menu from bottom-left.
+** You can freely answer - as long as you are not logged in, nothing will be submitted. Even after logging in you can disable uploading answers.
+* Is quest disappearing after being solved?
+
+# Open a pull request
+
+[Pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) is a submission of proposed changes to the source code.
+
+You can put into the description of PR something like "fixes #1234" to [mark it as fixing this issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword). It can go into a commit message or PR description ([not its title](https://github.com/streetcomplete/StreetComplete/discussions/2917), it is a known GitHub bug).
+
+If you are stuck on something, need some help or guidance and you are willing and able to continue after solving the problem - you can open a pull request in an incomplete state and mention the blocker.
+
+You can see [already submitted pull requests](https://github.com/streetcomplete/StreetComplete/pulls?q=is%3Apr+) to see how this process works in practice.
+
+# Future
+
+After opening a pull request it will be reviewed and you will likely be asked to make some changes. This is normal and happens also with pull requests submitted by experienced contributors.
+
+Changes typically include improving code style, tweaking phrasing and quest settings.
+
+You are also welcome to help with reviewing other PRs - different people have different strengths, there are active reviewers who help with code style, there are some native speakers of English, some with deep knowledge of OSM tagging schemes or deep knowledge of how StreetComplete works.
+
+After the PR was finished it will be merged before the beta release of the next version. This way it can be additionally tested with a wider audience before release to all and translators can [translate text into other languages](CONTRIBUTING.md#translating-the-app).
```

Comment:
```suggestion
After the PR is finished it will be merged before the beta release of the next version. This way it can be additionally tested with a wider audience before release to all and translators can [translate text into other languages](CONTRIBUTING.md#translating-the-app).
```
or
```suggestion
After the PR reviews are finished it will be merged before the beta release of the next version. This way it can be additionally tested with a wider audience before release to all and translators can [translate text into other languages](CONTRIBUTING.md#translating-the-app).
```